### ISSUE TITLE
Request inference script for logps data

### ISSUE THREAD

**Issue by tibetgao** [RLHF-V/RLHF-V#16]
Hi there,
Thanks for your awesome work on RLHF_V, I have noticed that you have provided weights for a pretrained model to generate logps, may I have the code for implementing this model to produce data?

Cheers!

**Comment by tibetgao** [comment]
As an follow up, will you please indicate the detail of fine-tuning the reward model so that it can work with othere decoders?

**Comment by yiranyyu** [comment]
The reward model is indeed the model being trained as policy model in DDPO (also in DPO). To produce the candidate responses to be annotated, we simply query the model with multimodal instructions without more complex strategies.
